interfaces/webrtc.idl

```diff
@@ -626,16 +625,6 @@ dictionary RTCStats {
   required DOMString id;
 };
 
-[Exposed=Window]
-interface RTCStatsEvent : Event {
```

Comment:
@henbos what's going on here?

Reply:
We removed the onstatsended event (as per TPAC discussions); stats refactoring has made the ”track” stats obsolete, and as such, the need to fire this event when replaceTrack occurs

Reply:
Alright, not a mistake, that's all I wanted to check.